dspace-api/src/main/java/org/dspace/xmlworkflow/XmlWorkflowServiceImpl.java

```diff
@@ -1041,4 +1058,62 @@ protected void notifyOfReject(Context c, XmlWorkflowItem wi, EPerson e,
     public String getMyDSpaceLink() {
         return ConfigurationManager.getProperty("dspace.url") + "/mydspace";
     }
+
+    public void createMessage(Context context, Item item, String description, String subject)
```

Comment:
Add Javadocs please

Reply:
removed as related to the messaging feature to be discussed in a separate PR